### PR TITLE
Feature/add component test

### PR DESCRIPTION
**Pull request by redhug1**
### What

Add section to docs:
To complete the move from nomad-job to nomad-pipeline

### How to review

read docs

### Who can review

anyone, but ideally Neill.

### CODE REVIEW

File: Adding Component Testing - HOWTO.md [hunk 1]
```diff
@@ -405,3 +405,74 @@ coverage.txt
...
+
+    And follow the instructions in what the above outputs.
+
+    (this creates a `.flyrc` file in your home folder that has a target for `dp-ci`)
+
+    The `make bootstrap` command takes less than a minute to run and in concourse you should see the new `develop-component`, thus:
+
+    ![](docs/img/develop-component.png)
+
+    Then click on develop-component then on the top right hand side, click (attached) to run the pipeline.
```
Comment (outsidecoder) [inline Adding Component Testing - HOWTO.md:470]: You could describe the button as a recycle or refresh symbol.

File: Adding Component Testing - HOWTO.md [hunk 2]
```diff
@@ -405,3 +405,74 @@ coverage.txt
...
+    
+    ![](docs/img/run-again.png)
+
+    For more information on how the task is running, or has run, click task: component to show the trace:
+    
+    ![](docs/img/run-progress.png)
+
+    This should all happen in less than 5 minutes.
```
Comment (outsidecoder) [inline Adding Component Testing - HOWTO.md:478]: less than 5 minutes, depending on how large your test is.

File: Adding Component Testing - HOWTO.md (hunk 1 above)
Comment (redhug1) [reply Adding Component Testing - HOWTO.md:470]: Done

File: Adding Component Testing - HOWTO.md (hunk 2 above)
Comment (redhug1) [reply Adding Component Testing - HOWTO.md:478]: Done

**Review by outsidecoder — APPROVED** [review]

**Pull request merged**